src/components/icon/icons/index.js

```diff
@@ -1,84 +1,85 @@
-import Tag from './tag';
-import TickingClock from './tickingClock';
-import Trash from './trash';
-import VerticalEllipsis from './verticalEllipsis';
+import AddMoney from "./addMoney";
+import Advice from "./advice";
+import Apartment from "./apartment";
+import ArrowDown from "./arrowDown";
+import ArrowLeft from "./arrowLeft";
+import ArrowRight from "./arrowRight";
+import ArrowUp from "./arrowUp";
+import ArrowUpDown from "./arrowUpDown";
+import Balloon from "./balloon";
+import Bank from "./bank";
+import Bell from "./bell";
+import Calendar from "./calendar";
+import Car from "./car";
+import Cart from "./cart";
+import Checkmark from "./checkmark";
+import CheckmarkLarge from "./checkmarkLarge";
+import ChevronDown from "./chevronDown";
+import ChevronUp from "./chevronUp";
+import CircleArrow from "./circleArrow";
+import CircleSlash from "./circleSlash";
+import Clock from "./clock";
+import Close from "./close";
+import Cloud from "./cloud";
+import Drop from "./drop";
+import Edit from "./edit";
+import Ellipse from "./ellipse";
+import Ellipsis from "./ellipsis";
+import ExclamationPoint from "./exclamationPoint";
+import ExclamationPointCircle from "./exclamationPointCircle";
+import Expand from "./expand";
+import Feed from "./feed";
+import File from "./file";
+import Film from "./film";
+import Filter from "./filter";
+import FloppyDisk from "./floppyDisk";
+import Folder from "./folder";
+import Gear from "./gear";
+import Globe from "./globe";
+import GraphArea from "./graphArea";
+import GraphCandlestick from "./graphCandlestick";
+import GraphLine from "./graphLine";
+import GraphVolume from "./graphVolume";
+import Heart from "./heart";
+import Hexagon from "./hexagon";
+import House from "./house";
+import Industry from "./industry";
+import Info from "./info";
+import Key from "./key";
+import LightBulb from "./lightBulb";
+import LightningBolt from "./lightningBolt";
+import Lock from "./lock";
+import Mail from "./mail";
+import MailOpen from "./mailOpen";
+import Market from "./market";
+import MedKit from "./medKit";
+import Mobile from "./mobile";
+import News from "./news";
+import OrderDepth from "./orderDepth";
+import Portfolio from "./portfolio";
+import Print from "./print";
+import Profile from "./profile";
+import Questionmark from "./questionmark";
+import SearchIcon from "./search";
+import SendMessage from "./sendMessage";
+import Share from "./share";
+import SignOut from "./signOut";
+import SocialFacebook from "./socialFacebook";
+import SocialGplus from "./socialGplus";
+import SocialInstagram from "./socialInstagram";
+import SocialTwitter from "./socialTwitter";
+import SocialYouTube from "./socialYouTube";
+import SolidArrowDown from "./solidArrowDown";
+import SolidArrowUp from "./solidArrowUp";
+import Star from "./star";
+import Tag from "./tag";
+import TickingClock from "./tickingClock";
+import Trash from "./trash";
+import VerticalEllipsis from "./verticalEllipsis";
```

Comment:
Why double quotes? :(

Reply:
@mlenser My answer could be "because I didn't pay attention" or "because Prettier config was broken" :). I made a new PR where I fix it, if we merge that one first and then this one we'll be good.